app/views/home/maintainers.html.erb

```diff
@@ -114,22 +114,22 @@
       <div class="col-sm-6 float-left mb-4 mb-md-0 pr-sm-3 pr-lg-4">
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars1.githubusercontent.com/u/1162160?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote of some wisdom from a maintainer”</p>
-            <footer class="alt-text-small">Jane Smith, Homebrew</footer>
+            <p class="alt-mono-font alt-text-small">The thing that most open source projects lack is not money, but time. Many maintainers have full-time jobs and can't realistically cut back their hours. What almost all projects need is more contributors. Open Source Friday is a perfect way for companies to support the open source ecosystem they depend on in a meaningful way.</p>
+            <footer class="alt-text-small">Rich Harris (<a href="https://github.com/Rich-Harris">@Rich-Harris</a>), <a href="https://github.com/rollup/rollup">Rollup</a> and <a href="https://github.com/sveltejs/svelte">Svelte</a> maintainer</footer>
           </blockquote>
         </div>
       </div>
       <div class="col-sm-6 float-left pl-sm-3 pl-lg-4">
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday2.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars3.githubusercontent.com/u/54418?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Another quote that talks about how Open Source Friday improved my project.”</p>
-            <footer class="alt-text-small">Juan Perez, Strap</footer>
+            <p class="alt-mono-font alt-text-small">Once you are a maintainer on a project that others rely on, open source goes from "sharing the code" to "sharing the maintenance". As we all know, "maintenance" takes time which is most effectively spent in a continuous block to help make sure you can finish that e.g. code review before you start to lose context.</p>
+            <footer class="alt-text-small">Brett Cannon (<a href="https://github.com/brettcannon">@brettcannon</a>), <a href="https://github.com/python/cpython">Python core</a> developer</footer>
```

Comment:
@brettcannon Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!